api/app/Generators/UserExcelGenerator.php

```diff
@@ -142,140 +224,668 @@ public function generate(): self
+            $careerObjectiveWorkStreams->join(', '), // career objective - Work streams
+            $careerObjectiveDepartments->join(', '), // career objective - Departments
+            $employeeProfile ? $employeeProfile->career_objective_additional_information ?? '' : '', // Career objective - Additional information
+            $employeeProfile ? $employeeProfile->career_planning_about_you ?? '' : '', // Career planning - About you
+            $employeeProfile ? $employeeProfile->career_planning_learning_goals ?? '' : '',  // Career planning - Learning goals
+            $employeeProfile ? $employeeProfile->career_planning_work_style ?? '' : '', // Career planning - Work style
+            $appliedPools->join(', '), // Digital talent processes
+            $offPlatformProcesses->join(', '), // Off-platform processes
+        ];
+    }
+
+    /**
+     * Add experiences to Career Experience sheet
+     */
+    private function generateExperiences(Worksheet $sheet, array $userIds, string $experienceClass, int &$currentRow): void
+    {
+        // get experiences for selected user
+        $experienceClass::whereIn('user_id', $userIds)
+        // load data relationships
+            ->with(['user', 'userSkills.skill'])
+            // load data for work experiences
+            ->when($experienceClass === WorkExperience::class, function ($query) {
+                // work experiences need additional data
+                $query->with(['department', 'classification', 'workStreams']);
+            })
+            ->chunk(200, function ($experiences) use ($sheet, &$currentRow) {
+                foreach ($experiences as $exp) {
+                    // convert each experience to a excel row
+                    $rowData = $this->buildExperienceRow($exp);
+                    $sheet->fromArray($rowData, null, sprintf('A%d', $currentRow));
+                    $currentRow++;
+                }
+            });
+    }
+
+    /**
+     * Get localized experience type for an experience
+     */
+    private function getExperienceType($experience): string
+    {
+        return match (true) {
+            $experience instanceof WorkExperience => $this->localizeEnum(ExperienceType::WORK->name, ExperienceType::class),
+            $experience instanceof EducationExperience => $this->localizeEnum(ExperienceType::EDUCATION->name, ExperienceType::class),
+            $experience instanceof AwardExperience => $this->localizeEnum(ExperienceType::AWARD->name, ExperienceType::class),
+            $experience instanceof CommunityExperience => $this->localizeEnum(ExperienceType::COMMUNITY->name, ExperienceType::class),
+            $experience instanceof PersonalExperience => $this->localizeEnum(ExperienceType::PERSONAL->name, ExperienceType::class),
+            default => '',
+        };
+    }
+
+    /**
+     * Build experiences row
+     */
+    private function buildExperienceRow($exp): array
+    {
```

Comment:
This method is not localizing any enum values, we will likely want to add that too.